test/e2e/kubectl.go

```diff
@@ -546,6 +548,35 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Proxy unix socket", func() {
+		It("should support --unix-socket=/path", func() {
```

Comment:
Nit: I would move this above under the existing `Describe()` for proxy server, since it is testing a different option of the same command.